doc/workflow/workflow.rst

```diff
@@ -167,21 +167,21 @@ We use a regression learner for the outcome variable net financial assets.
                         eta = 0.1, nrounds = 34)
 
 
-4. DML Specifications
+1. DML Specifications
 ---------------------
 
 In Step 4., we initialize and parametrize the model object which will later be used to perform the estimation.
 
-We initialize a DoubleMLPLR [link to docu] using the previously generated data-backend. Moreover, we specify the resampling
-(= the number of repetitions and folds for repeated cross-fitting [link to user guide section resampling]),
-the dml algorithm (DML1 vs. DML2 [link to algorithm section in user guide]) and the score function ("partialling out" or
-"IV-type") [link to user guide section on score].
+We initialize a `DoubleMLPLR <https://docs.doubleml.org/stable/api/generated/doubleml.DoubleMLPLR.html>`_ using the previously generated data-backend. Moreover, we specify the resampling
```

Comment:
Link to Py and R API docu?

Reply:
added in 06afc1d

Reply:
Maybe it's enough to highlight the code as in the user guide and later have a short block with references to the docu? For example as in here https://docs.doubleml.org/stable/guide/data_backend.html and https://docs.doubleml.org/stable/guide/models.html

Reply:
But you can leave this open and I'll fix it once you've integrated it in the dev docu? I think we'll have a final polishing step anyways

Reply:
Alright, sounds like a good plan.